index.html

```diff
@@ -17,18 +17,18 @@
         <a href="index.html">Juma</a>
       </div>
       <nav class="nav">
-        <ul>
-          <li><a href="#">Portfolio</a></li>
-          <li><a href="#">About Me</a></li>
-          <li><a href="#">Contact</a></li>
+        <ul class="nav-item">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About Me</a></li>
+          <li><a href="#contact">Contact</a></li>
         </ul>
```

Comment:
- [ OPTIONAL] :thinking: I think it would be better to make this list fit on the same line to meet the Figma design:

![Screenshot from 2023-01-20 22-12-56](https://user-images.githubusercontent.com/97834160/213797884-73ac4ccf-713d-481d-93fd-630de67eb14d.png)